sale_financial_risk/models/sale.py

```diff
@@ -8,6 +8,8 @@
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
+    partner_invoice_id = fields.Many2one(index=True)
```

Comment:
Add here a comment of why this is put:

```suggestion
    # index this field, as it's used in the compute of field ...
    partner_invoice_id = fields.Many2one(index=True)
```

Reply:
Done